src/comp/comp.py

```diff
@@ -24,48 +24,49 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [i.name for i in humans if i.name[0] == "D"]
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = []
+b = [i.name for i in humans if i.name[-1] == "e"]
 print(b)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with any letter between 'C' and 'G' inclusive.
 print("Starts between C and G, inclusive:")
-c = []
+# i.name for i in humans if i.name[0] == 
+c = [i.name for i in humans if (i.name[0] == "C" or i.name[0] == "D" or i.name[0] == "E" or i.name[0] == "F" or i.name[0] == "G")]
```

Comment:
You can do it more concisely with something like [h.name for h in humans if h.name[0] in "CDEFG"] :)